### ISSUE TITLE
Will it update to strapi v4?

### ISSUE THREAD

**Issue by kingchun1991** [edgarlr/magazine-api#1]

**Comment by edgarlr** [comment]
@kingchun1991 yes, is planned to be updated to v4, I just haven't had enoght time to do it myself, I'll probably start doing it in the next couple of weeks

**Comment by kingchun1991** [comment]
@edgarlr Anything I can help with?

I lost all data and haven't backed up trying to update Strapi from v3 and v4 in Heroku.

Turn out there is a backup a week for the free tier.

**Comment by edgarlr** [comment]
Sure! if you have the time to update this template yourself and open a PR that would be awesome, if not, no worries, I'll probably start working on later this week early next one!
